django/contrib/gis/admin/__init__.py

```diff
@@ -2,11 +2,12 @@
     HORIZONTAL, VERTICAL, AdminSite, ModelAdmin, StackedInline, TabularInline,
     autodiscover, register, site,
 )
+from django.contrib.gis.admin.admin import GISModelAdmin
 from django.contrib.gis.admin.options import GeoModelAdmin, OSMGeoAdmin
 from django.contrib.gis.admin.widgets import OpenLayersWidget
 
 __all__ = [
     'HORIZONTAL', 'VERTICAL', 'AdminSite', 'ModelAdmin', 'StackedInline',
     'TabularInline', 'autodiscover', 'register', 'site',
-    'GeoModelAdmin', 'OSMGeoAdmin', 'OpenLayersWidget',
+    'GeoModelAdmin', 'OSMGeoAdmin', 'GISModelAdmin', 'OpenLayersWidget',
```

Comment:
I'd move depracated classes to a separate line:
```suggestion
    'GISModelAdmin', 'OpenLayersWidget',
    # RemovedInDjango50Warning.
    'GeoModelAdmin', 'OSMGeoAdmin',
```